apps/pie-docs/.eleventy.js

```diff
@@ -5,13 +5,22 @@ const {
     plugins
 } = require('./src/_11ty');
 
+const litPlugin = require('@lit-labs/eleventy-plugin-lit');
+
+
 module.exports = eleventyConfig => {
     // Copy over img directory to dist directory.
     eleventyConfig.addPassthroughCopy({ 'src/assets/img': 'assets/img' });
 
     // Plugins
     plugins.addAllPlugins(eleventyConfig);
 
+    eleventyConfig.addPlugin(litPlugin, {
+        mode: 'worker',
+        componentModules: [
+            'src/assets/js/pie-components-imports.js',
+        ],
+    });
```

Comment:
please move all of this out into the plugins file all the other plugins are added in